### ISSUE TITLE
Wrong PHPDOC for Event `setAttendees()`

### ISSUE THREAD

**Issue by N-M** [microsoftgraph/msgraph-sdk-php#726]
If it is added like this
![image](https://user-images.githubusercontent.com/781417/144323896-a8001ac2-fbd7-4084-8ab3-a32ee56905bf.png)
Following exception will appear 

![image](https://user-images.githubusercontent.com/781417/144324049-e6e88ff3-ab94-488f-84aa-31b18e77249b.png)

Code below works but IDE and code quality checks are failing

![image](https://user-images.githubusercontent.com/781417/144323645-f4b2b5a2-104a-44db-b1a5-9251d236005a.png)

Doc param should be changed from 

`@param Attendee[] $val The attendees
`

to 

`@param Attendee $val The attendees`

**Comment by Ndiritu** [comment]
Thanks for raising this @N-M 

My apologies, we did not anticipate breaking code quality checks like PHPStan when we fixed the bug in the PHPDoc type, changing from `Attendee` to `Attendee[]`.

Ideally, to create an event, we should be passing a `Attendee[]` from the [docs](https://docs.microsoft.com/en-us/graph/api/resources/event?view=graph-rest-1.0#properties) but unfortunately we previously set the wrong PHPDoc type. Looks like you found a workaround by initialising an `Attendee` object with a list of Attendees which serializes to the expected collection of attendees hence `$event->setAttendees($outlookAttendees)` works.

The change to the PHPDoc types was made to cater for new developers starting on the SDK who would pass a single `Attendee` object to `setAttendees()` and end up getting a 400 since the API expects a collection.

The solution here seems to be to revert the PHPDoc change and mark this as a known issue on our v1 SDK with a workaround for developers new to this SDK. I shall discuss this with the rest of the team and share the final conclusion.
